src/System.Security.Cryptography.Algorithms/tests/PaddingModeTests.cs

```diff
@@ -57,6 +51,46 @@ private static void ValidatePaddingMode(PaddingMode paddingMode, int expectedPad
             }
         }
 
+        [Theory]
+        [InlineData(PaddingMode.ISO10126, 1, "46785BDE46622B92FF7C8EBB91508A")]
+        [InlineData(PaddingMode.ISO10126, 13, "E505A2")]
+        [InlineData(PaddingMode.ISO10126, 16, "")]
+        private static void ValidatePaddingMode_ISO10126(PaddingMode paddingMode, int expectedPaddingSize, string plainTextStr)
+        {
+            Assert.True(paddingMode == PaddingMode.ISO10126, "This tests only ISO10126 padding");
```

Comment:
Since the padding mode is in the name, maybe it should be removed from the theory/arguments list?